python/libucxx/libucxx/load.py

```diff
@@ -44,6 +44,11 @@ def _load_wheel_installation(soname: str):
 
 def load_library():
     """Dynamically load libucxx.so and its dependencies"""
+    try:
+        import librmm  # noqa: F401
```

Comment:
Just to be safe, this won't create a CUDA context upon import, will it?

Reply:
No, it won't. At the moment really all this is doing is making the underlying rapids-logger library available.

Reply:
Makes sense, I just want to ensure this remains usable for non-CUDA users if they have RMM installed but don't intend on using it.